spec/cypress/integration/notifications/AdminAccountCreatedEmail.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Admin Account Created Email
+  Email will be sent to the user when Admin creates their account
+
+  Scenario: Sending Email to newly Created Admin account
+    Given I am logged in as an "admin"
+    And I am on "admin index" page
+    When I click on "create admin button"
+    Then I should be on "admin creation" page
+
+    When I type "Joe Wick" into "name input"
+    And I type "new-admin@example.com" into "email input"
+    And I click the submit button
+    And "main" should contain "Joe Wick"
+    And "main" should contain "new-admin@example.com"
+    And I click the submit button
+    Then "notification banner" should contain "User added"
+    And An Admin account created email will be sent to the email "new-admin@example.com"
```

Comment:
What's the reason this has been added as a new test instead of being added to the existing test?

Reply:
I can move it to Admin Management feature and add it as last step - main reason was to have explicit Notification tests that tell a story from the perspective of sending notifications. I guess the key question is how to achieve a balance - should we have separate features each telling about different aspect or should we focus on long features that tests everything inbetween.

Reply:
We already have multi-domain tests as writing specific tests for everything would slow the tests down a lot

This test takes three seconds to run, according to the github action

Reply:
Moved to Account Management